src/pybind/mgr/mirroring/fs/snapshot_mirror.py

```diff
@@ -745,25 +745,39 @@ def daemon_status(self, filesystem):
                 fspolicy = self.pool_policy.get(filesystem, None)
                 if not fspolicy:
                     raise MirrorException(-errno.EINVAL, f'filesystem {filesystem} is not mirrored')
-                daemons = {}
+                daemons = []
                 sm = self.mgr.get('service_map')
                 daemon_entry = sm['services'].get('cephfs-mirror', None)
-                if daemon_entry:
-                    for daemon_key in daemon_entry['daemons']:
-                        try:
-                            daemon_id = int(daemon_key)
-                            daemon_status = self.mgr.get_daemon_status('cephfs-mirror', daemon_key)
-                            if not daemon_status:
-                                # temporary, should get updated soon
-                                log.debug(f'daemon status not yet availble for daemon_id {daemon_id}')
-                                continue
-                            try:
-                                daemons[daemon_id] = json.loads(daemon_status['status_json'])
-                            except KeyError:
-                                # temporary, should get updated soon
-                                log.debug(f'daemon status not yet available for daemon_id {daemon_id}')
-                        except ValueError:
-                            pass
+                log.debug(f'daemon_entry: {daemon_entry}')
+                for daemon_key in daemon_entry.get('daemons', []):
+                    try:
+                        daemon_id = int(daemon_key)
```

Comment:
After thought, this is just the `daemon_name` right? It just happens to be an ID in this case.

Reply:
right, the RADOS instance id.